website/sidebars.json

```diff
@@ -1,15 +1,15 @@
 {
   "docs": {
     "Prerequisites": ["prereqs", "prereqs-ubuntu", "prereqs-windows"],
-    "Javascript Client SDK": ["loom-js-quickstart", "web3js-loom-provider-truffle"],
+    "Javascript Client SDK": ["loom-js-quickstart", "web3js-loom-provider-truffle", "json-rpc-methods"],
```

Comment:
my thought is we should have an advanced or internal section to the docs

Reply:
Yeah probably is a good idea to put into a more advanced area, any suggestion?